api/src/handlers/scrapePosts/index.js

```diff
@@ -0,0 +1,20 @@
+const scrapePosts = require('./scrapePosts.js');
+const Db = require('../../db/db.js')
+const FeedManager = require('../../rss/feedManager.js')
+const Parser = require('rss-parser');
+
+module.exports.handler = (event, context, callback) => {
```

Comment:
Can't you change this callback to a `new Promise`?

Reply:
the `callback` is a hook into aws lambda (not sure we can modify that directly into a promise - other than wrapping it in a `resolve` / `reject`) + all the docs use this form so thought it was better to stick to the conventions